build.gradle.kts

```diff
@@ -13,13 +13,13 @@ if (System.getenv("RELEASE") != "true") {
 
 autoConfigure {
     java {
-        languageVersion.set(JavaLanguageVersion.of(8))
+        languageVersion.set(JavaLanguageVersion.of(11))
         vendorName.set("Cloudflight")
     }
     kotlin {
-        // we let the plugin itself by dependent on Kotlin 1.5.31 as this is the version currently being used
+        // we let the plugin itself by dependent on Kotlin 1.6.21 as this is the version currently being used
         // by the gradle runtime of version 7.4.1
-        kotlinVersion.set("1.5.31")
+        kotlinVersion.set("1.6.21")
```

Comment:
But according to this: https://docs.gradle.org/7.4.2/userguide/compatibility.html#kotlin Gradle 7.4.1 only bundles Kotlin 1.5 (and Gradle plugins written in Kotlin should only target Kotlin 1.4 to begin with)